src/parsers/protobuf/main.ts

```diff
@@ -157,13 +157,18 @@ export default async function parseProtobuf(
   }
 
   const rootSector = sectors['0/'];
+  for (const primitiveGroup of rootSector.traversePrimitiveGroups()) {
+    primitiveGroup.sort();
+  }
+
   mergeInstancedMeshes(rootSector, sceneStats);
   for (const sector of rootSector.traverseSectors()) {
     sector.mergedMeshGroup.createTreeIndexMap();
     sector.instancedMeshGroup.createTreeIndexMap();
 
     sceneStats.numSectors++;
     sector.primitiveGroups.forEach(primitiveGroup => {
+      // @ts-ignore
```

Comment:
Why did we need a `ts-ignore` here? Could we fix it instead? If not, at least add the exception to the ignore statement.

Reply:
Oh that's actually unnecessary. Removed.